src/deriv_api/DerivAPIBasic.js

```diff
@@ -76,6 +76,7 @@ export default class DerivAPIBasic extends DerivAPICalls {
         this.pendingRequests       = {};
         this.expect_response_types = {};
         this.subscription_manager  = new SubscriptionManager(this);
+        this.timeoutObject         = '';
```

Comment:
Variable name should be in `snake_case` and I think `reconnect_timeout` is a better name for this variable